src/subdomains/generic/gs/gs.service.ts

```diff
@@ -93,6 +96,13 @@ export class GsService {
 
     return {
       userData,
+      documents: [
+        ...(await this.documentStorageService.listUserFiles(userData.id)),
+        ...(await this.documentStorageService.listSpiderFiles(
+          userData.id,
+          userData.accountType !== AccountType.PERSONAL,
+        )),
```

Comment:
If `userData.accountType !== AccountType.PERSONAL` you need to get documents for user and organization.